config/locales/en.yml

```diff
@@ -896,6 +896,7 @@ en:
     not_started: Not started
     permanent_failure: Permanent failure
     posted: Posted
+    printing: Printing
```

Comment:
@cdccollins it looks like in the NeighbourLetter model the `accepted` status is also called `printing` and then it looks that value up in the translations, not `accepted` — can I just double-check with you that that makes sense?